sceptre/cli/list.py

```diff
@@ -127,3 +126,35 @@ def list_change_sets(ctx, path, url):
 
     for response in responses:
         write(response, context.output_format)
+
+
+@list_group.command(name="stacks")
+@click.argument("path")
+@click.pass_context
+@catch_exceptions
+def list_stacks(ctx, path):
+    """
+    List stack names and command paths for stacks in a
+    stack group.
+    \f
+
+    :param path: Path to execute the command on.
+    :type path: str
+    """
+    context = SceptreContext(
+        command_path=path,
+        project_path=ctx.obj.get("project_path"),
+        user_variables=ctx.obj.get("user_variables"),
+        output_format=ctx.obj.get("output_format"),
+        options=ctx.obj.get("options"),
+        ignore_dependencies=ctx.obj.get("ignore_dependencies")
+    )
+
+    plan = SceptrePlan(context)
+
+    if context.output_format == "json":
+        output = {f"{stack.name}.yaml: {stack.external_name}" for stack in plan.all_stacks}
+    else:
+        output = [f"{stack.name}.yaml: {stack.external_name}" for stack in plan.all_stacks]
```

Comment:
I'm not sure why you did this. Why not just always make it a dictionary comprehension? 

This is all I wanted: 
```suggestion
    output = {f"{stack.name}.yaml: {stack.external_name}" for stack in plan.all_stacks
```